src/Cake.Common/Tools/NuGet/Restore/NugetRestoreSettings.cs

```diff
@@ -73,6 +73,12 @@ public sealed class NuGetRestoreSettings : ToolSettings
         /// <value>The version of MSBuild to be used with this command.</value>
         public NuGetMSBuildVersion? MSBuildVersion { get; set; }
 
+        /// <summary>
+        /// Gets or sets the path of MSBuild to use.
+        /// This setting takes precedence over <c>-MSBuildVersion</c> and requires NuGet V4 or later.
+        /// </summary>
+        public DirectoryPath MSDirectoryPath { get; set; }
```

Comment:
Change to `MSBuildPath` to match parameter.

Reply:
Done.